src/net.cpp

```diff
@@ -2834,6 +2883,8 @@ void CConnman::ThreadOpenAddedConnections()
         // Retry every 60 seconds if a connection was attempted, otherwise two seconds
         if (!interruptNet.sleep_for(std::chrono::seconds(tried ? 60 : 2)))
             return;
+        // See if any reconnections are desired.
+        PerformReconnections();
```

Comment:
Wouldn't this wait for 60 seconds? (context: https://github.com/bitcoin/bitcoin/pull/28331#issuecomment-1718407366)